### PR TITLE
[Fuchsia] Support per app present latency tracing

### PR DESCRIPTION
**Pull request by chaopeng**
Bug: http://b/323408867

### CODE REVIEW

File: shell/platform/fuchsia/flutter/flatland_connection.h [hunk 1]
```diff
@@ -105,6 +105,7 @@ class FlatlandConnection final {
   std::vector<zx::event> acquire_fences_;
   std::vector<zx::event> current_present_release_fences_;
   std::vector<zx::event> previous_present_release_fences_;
+  std::string debug_label_;
```
Comment (uysalere) [inline shell/platform/fuchsia/flutter/flatland_connection.h:108]: Add const

**Review by uysalere — APPROVED** [review]

**Comment by chinmaygarde** [comment]
Ping @chaopeng. Are you blocked on this?

**Comment by chaopeng** [comment]
> Ping @chaopeng. Are you blocked on this?

Yes, the parent cl on fuchsia is not land yet

**Comment by chinmaygarde** [comment]
The CL seems to have landed. Can we move this along?

**Pull request merged**